Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [workspace]
 members = [ 
     "crates/web5", 
-    "bindings/uniffi",
+    "bind/uniffi",
```

Comment:
Hmm almost liking bindings here more actually, but I wonder if there is an official 'rust' way to do it

Reply:
Yeah I'm with you, switched back to `bindings/`